cmd/libs/go2idl/client-gen/generators/client_generator.go

```diff
@@ -41,9 +41,36 @@ func NameSystems() namer.NameSystems {
 		"Endpoints": "Endpoints",
 	}
 	lowercaseNamer := namer.NewAllLowercasePluralNamer(pluralExceptions)
+
+	publicNamer := &ExceptionNamer{
+		Exceptions: map[string]string{
+		// these exceptions are used to deconflict the generated code
+		// you can put your fully qualified package like
+		// to generate a name that doesn't conflict with your group.
+		// "k8s.io/apis/events/v1alpha1.Event": "EventResource"
+		},
+		KeyFunc: func(t *types.Type) string {
+			return t.Name.Package + "." + t.Name.Name
+		},
+		Delegate: namer.NewPublicNamer(0),
+	}
+	privateNamer := &ExceptionNamer{
+		Exceptions: map[string]string{
+		// you can put your fully qualified package like
```

Comment:
nit - can you make those comments exactly the same? Current the order of lines is slightly different.